cmd/kpod/push.go

```diff
@@ -117,8 +116,7 @@ func pushCmd(c *cli.Context) error {
 			SignBy:           signBy,
 		},
 	}
-	if !c.Bool("quiet") {
-		options.ReportWriter = os.Stderr
-	}
-	return images.PushImage(srcName, destName, options)
+
+	runtime.GetConfig().SignaturePolicyPath = c.String("signature-policy")
```

Comment:
This won't modify the signature policy in the runtime, which is immutable - pass WithSignaturePolicy when creating the runtime to do that

Reply:
Fixed.  Added to CopyOptions

Reply:
Do we need it in CopyOptions? We should be able to use the one in the runtime instead, I think. Is it important that we be able to override the default signature policy settings for pulls?